src/java/picard/sam/markduplicates/MarkDuplicates.java

```diff
@@ -359,6 +412,8 @@ private void buildSortedReadEndLists(final boolean useBarcodes) {
             this.libraryIdGenerator = new LibraryIdGenerator(header);
         }
 
+        String duplicateSetName = null;
```

Comment:
I like `duplicateSetQueryName` better.  Perhaps even `duplicateQueryName` since duplicate sets aren't mentioned anywhere else in this code.